mozjs/js/rust/build.rs

```diff
@@ -25,7 +26,16 @@ fn build_jsglue_cpp() {
 
 /// Find the public include directory within our mozjs-sys crate dependency.
 fn get_mozjs_include_dir() -> path::PathBuf {
-    let entries = glob::glob("./target/*/build/mozjs_sys-*/out/dist/include")
+    let out_dir = env::var("OUT_DIR")
```

Comment:
The commit message mentions the need for upstreaming. Has this happened? Is this still WIP?